packages/toolboxes/cosmos/src/thorchainUtils/types/client-types.ts

```diff
@@ -81,6 +81,7 @@ export type TransferTxParams = {
   toAddress: string;
   assetValue: AssetValue;
   memo?: string;
+  fee?: string;
```

Comment:
We should think about this a bit more - might want different type here - will release for now